client/my-sites/people/people-list-item/index.jsx

```diff
@@ -44,40 +45,58 @@ class PeopleListItem extends React.PureComponent {
 		);
 	};
 
+	getCardLink = () => {
+		const { invite, site, type, user } = this.props;
+		const editLink = this.canLinkToProfile() && `/people/edit/${ site.slug }/${ user.login }`;
+		const inviteLink = invite && `/people/invites/${ site.slug }/${ invite.invite_key }`;
+
+		return type === 'invite' ? inviteLink : editLink;
+	};
+
+	renderInviteStatus = () => {
+		const { invite, translate } = this.props;
+		const { is_pending: isPending } = invite;
+		const statusClasses = {
+			'is-pending': isPending,
+		};
+		const className = classNames( 'people-list-item__invite-status', statusClasses );
+
+		return (
+			<div className={ className }>
+				{ ! isPending && <Gridicon icon="checkmark" size={ 18 } /> }
+				{ isPending ? translate( 'Pending' ) : translate( 'Accepted' ) }
+				{ isPending && (
+					<Button className="people-list-item__invite-resend" compact>
+						{ translate( 'Resend Invite' ) }
+					</Button>
+				) }
+			</div>
+		);
+	};
+
 	render() {
+		const { className, invite, onRemove, translate, type, user } = this.props;
 		const canLinkToProfile = this.canLinkToProfile();
 		const tagName = canLinkToProfile ? 'a' : 'span';
+		const isInvite = invite && 'invite' === type;
 
 		return (
 			<CompactCard
-				{ ...omit(
-					this.props,
-					'className',
-					'user',
-					'site',
-					'isSelectable',
-					'onRemove',
-					'moment',
-					'numberFormat',
-					'translate'
-				) }
```

Comment:
Nice job getting rid of this.

Reply:
Yeah -- it wasn't used anywhere and seemed like it'd be hard to maintain. I think being explicit about props is a better practice.